beacon-chain/chaintest/backend/fork_choice_test_format.go

```diff
@@ -1,37 +1,37 @@
 package backend
 
-// ChainTest --
-type ChainTest struct {
+// ForkChoiceTest --
```

Comment:
Why do we name this `ForkChoice`? I thought this is more related to per slot processing and per epoch processing. Maybe per epoch processing should be its own set of tests

Reply:
1.) per slot processing test
2.) per epoch processing test
3.) fork choice test
4.) shuffle test

Reply:
We can think about that more - it would be nice to have a single place where you can a full state transition test with a yaml config which is what I've been aiming for with these PRs.